### PR TITLE
starlight files uploaded

### PR DESCRIPTION
**Pull request by KiSooLee**
spec file and patch file for starlight are uploaded

### CODE REVIEW

**Comment by ghost** [comment]
Dear @KiSooLee, few comments:
1) New externals should always go to the development cycle, which right now is the 8_0_X
2) I though that we would be using Starlight from the external repository that we created here: cms-externals/starlight (which means changing the source and no need for the patch as it's already patched in the repository, see for example: https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_8_0_X/stable/pythia8.spec)
3) The spec file you provided isn't really in line with the spec files we have in this repository. (eg remove unwanted sections: license, doc...)
4) The spec file is still missing some piece (eg line 33), have you actually tested this?
5) The spec file `cmssw-tool-conf.spec` should also be updated to include `starlight.spec`
6) You should also provide a "toolconf" that will enable the use of this generator in CMSSW. See, for example, https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_8_0_X/stable/pythia8-toolfile.spec.

**Comment by KiSooLee** [comment]
Ok.
Let me try again in the CMSSW_8_0_X.
